src/GregClient/GregClient.csproj

```diff
@@ -54,8 +55,36 @@
       <Pack>True</Pack>
       <PackagePath>\</PackagePath>
     </None>
+    <Compile Include="..\AssemblyInfoGenerator\AssemblyInfo.cs" Link="Properties\GlobalVersionInfo.cs" />
   </ItemGroup>
-  <ItemGroup Label="Compile time reference assemblies">
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="All" />
```

Comment:
so this doesn't work?

Reply:
Actually I do not know.
I will put it back